externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/result/ResultSetUtil.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.engine.flink.result;
+
+import org.apache.flink.table.api.DataTypes
+import org.apache.flink.table.api.ResultKind
+import org.apache.flink.table.catalog.Column
+import org.apache.flink.types.Row
+
+/** Utility object for building ResultSet. */
+object ResultSetUtil {
+
+  /**
+   * Build a ResultSet with a column name and a list of String values.
```

Comment:
Adds the parameters comments.

Reply:
I think the current comments are fine for an internal utility. But there's no harm in adding more comments, so I'm OK with this.